releases/2022.3/2022.3.md

```diff
@@ -0,0 +1,150 @@
+---
+title: 2022.3
+parent: Releases
+---
+
+<div align="center">
+  <h1>Kytos-ng 2022.3</h1>
+  <a href="https://github.com/kytos-ng/"><img src="/assets/images/kytosng_logo_color.png" width="80%" height="80%"></a>
+</div>
+
+<div align="center">
+  <a href="https://www.amlight.net"><img src="https://amlight.net/wp-content/uploads/2016/02/AmLlightExP_s.jpg"></a>
+</div>
+
+Kytos-ng[^1] version `2022.3` was released in December
+
+## Objectives
+
+This version had these major objectives:
+
+- `mef_eline` should be able to create EVCs with constrained paths and service level prioritization
+- Specify how INT will be integrated and supported via NoviFlow experimenter actions
+- `maintenance` should stabilize its v1 API and integrate with MongoDB 
+- Support NoviFlow switches on end-to-end tests
+- End-to-end tests should have fewer false positives and support retries
+- `flow_stats` should stabilize its v1 API
+- Drop support for OpenFlow 1.0
+- Support REST endpoints user authentication with MongoDB
+- Upgrade Kytos-ng core dependencies to leverage upstream fixes, enhancements and be able to leverage more asyncio functionalities
+- All Kytos-ng NApps should have at least 85% of unit test coverage
+
+In order to achieve the objectives of this version, the following enhancements were refined and delivered:
+
+### 1. `mef_eline` should be able to create EVCs with constrained paths
+
+`mef_eline` now supports EVC creation with path constraints that can be applied to a dynamic primary path or secondary dynamic failover paths. The following link metadata contrainsts can be used if they're configured: `bandwidth`, `utilization`, `priority`, `reliability`, `delay` and `ownership`. In addition other path attributes such as total accumulated cost, desired links and undesired links are also supported. The parameters are documented on [mef_eline OpenAPI specification](https://kytos-ng.github.io/api/mef_eline.html#operation/create_circuit).
+
+In addition, `mef_eline` provides service level prioritization to allow EVCs to be handled based on their descending `service_level` value during convergence and consistency checks.
+
+### 2. Specify how INT will be integrated and supported via NoviFlow experimenter actions
+
+A new `telemetry` has been specified on blueprint [EP031](https://github.com/kytos-ng/kytos/blob/master/docs/blueprints/EP031.rst), which will be responsible for enabling INT on EVCs. NoviFlow switches will be supported via experimenter actions that been implemented by the `noviflow` NApp, and the functionality will be optionally enabled when an EVC is being created. The `telemetry` NApp is expected to be available in the `2023.1` release.
+
+### 3. `maintenance` stabilization v1 API and integrate with MongoDB 
+
+**TODO** fill out what will be feasible to be delivered, ask David to update this paragraph
```

Comment:
FYI @Ktmi, if you could help out to summarize and update this document later that'd be great, thanks.